tests/chainerx_tests/unit_tests/routines_tests/test_reduction.py

```diff
@@ -247,3 +259,27 @@ def test_log_softmax_invalid(device, a_shape, axis, dtype):
     a = array_utils.create_dummy_ndarray(chainerx, a_shape, dtype)
     with pytest.raises(chainerx.DimensionError):
         return chainerx.log_softmax(a, axis=axis)
+
+
+@op_utils.op_test(['native:0'])
+@chainer.testing.parameterize_pytest(
+    'in_dtypes,out_dtype', math_utils.in_out_dtypes_math_functions)
```

Comment:
`chainerx.cumsum` should use the same dtype-rule as that of `chainerx.sum`.
https://github.com/chainer/chainer/blob/bc3dbef7eec8b77c72a7844844ee2401e1b026e1/tests/chainerx_tests/unit_tests/routines_tests/test_reduction.py#L54-L67.

Reply:
How about defining this list as `_in_out_dtypes_sum` and use it?